lib/babe/build_test.go

```diff
@@ -83,6 +85,45 @@ func addAuthorshipProof(t *testing.T, babeService *Service, slotNumber, epoch ui
 	babeService.slotToProof[slotNumber] = outAndProof
 }
 
+func createTestExtrinsic(t *testing.T, rt runtime.Instance, genHash common.Hash, nonce uint64) types.Extrinsic {
```

Comment:
This func looks the same as func in dot/core/messages_test.go and in lib/runtime/wasmer/exports_test.go, I wonder if we can make one of them public and re-use it?

Reply:
I'd rather not make it public and import one package into the other but I'll move it to utils

Reply:
nevermind, that causes a circular import. I'll think about how to do this